helper.js

```diff
@@ -149,8 +149,8 @@ function attributeEntry(el, label, value) {
 
 function createIframe(opt, width, height) {
   el = document.createElement("iframe")
-  width = typeof width !== 'undefined' ? width : '525px';
-  height = typeof height !== 'undefined' ? height : '350px';
+  width = typeof width !== 'undefined' ? width : '525px'
```

Comment:
can we set this to 100% instead of 525px? I just tested it, looks the same but should be more flexible in case on some devices the sidebar is narrower.